src/Http/Http/src/Features/FormFeature.cs

```diff
@@ -80,10 +81,16 @@ public bool HasFormContentType
         }
     }
 
+    internal bool HasInvalidAntiforgeryValidationFeature => ResolveHasInvalidAntiforgeryValidationFeature();
```

Comment:
Not sure if it's because of the layering that we need to do it this way, but an alternative way to achieve this is to wrap the FormFeature and make the check in the wrapper. That will help keep the concerns separated.